L1Trigger/L1TGEM/interface/ME0StubAlgoChamber.h

```diff
@@ -0,0 +1,21 @@
+#ifndef L1Trigger_L1TGEM_ME0StubAlgoChamber_H
+#define L1Trigger_L1TGEM_ME0StubAlgoChamber_H
+
+#include "L1Trigger/L1TGEM/interface/ME0StubAlgoSubfunction.h"
+#include "L1Trigger/L1TGEM/interface/ME0StubAlgoPartition.h"
+#include "L1Trigger/L1TGEM/interface/ME0StubAlgoMask.h"
+#include <vector>
+#include <cstdint>
+#include <algorithm>
+
+namespace l1t {
+  namespace me0 {
+    std::vector<std::vector<ME0StubPrimitive>> cross_partition_cancellation(
```

Comment:
function names should be formatted like `crossPartitionCancellation` (also code rule 2.8 http://cms-sw.github.io/cms_coding_rules.html). please propagate this change to all files in the PR.